File: src/main/java/io/junseok/mealmate/domain/restaurant/dto/response/RestaurantDetailInfo.java
package io.junseok.mealmate.domain.restaurant.dto.response;

import io.junseok.mealmate.domain.restaurantmenu.dto.response.MenuInfo;
import java.time.LocalDateTime;
import java.util.List;
import lombok.Builder;

@Builder
public record RestaurantDetailInfo(
    String restaurantName,
    String restaurantImageUrl,
    String restaurantType,
    int likeCount,
    double grade,
    LocalDateTime openAt,
    LocalDateTime closeAt,
    String restaurantNotice,
    List<MenuInfo> menuList) {

}

File: src/main/java/io/junseok/mealmate/domain/restaurant/dto/response/RestaurantDetailInfo.kt
package io.junseok.mealmate.domain.restaurant.dto.response;

import io.junseok.mealmate.domain.restaurantmenu.dto.response.MenuInfo

data class RestaurantDetailInfo(
    val restaurantName: String,
    val restaurantImageUrl: String,
    val restaurantType: String,
    val restaurantTelNum: String,
    val location: String,
    val likeCount: Int,
    val grade: Double,
    val openAt: String,
    val closeAt: String,
    val restaurantNotice: String,
    val menuList: List<MenuInfo>
)
